src/App.vue

```diff
@@ -287,12 +296,27 @@ export default class App extends Mixins(
 
   /** Data for fee summary component. */
   private get feeFilingData (): Array<FilingDataIF> {
-    return this.getFilingData
-      ? [{ ...this.getFilingData,
-        futureEffective: this.getEffectiveDateTime.isFutureEffective,
-        priority: this.getStaffPaymentStep.staffPayment.isPriority,
-        waiveFees: (this.getStaffPaymentStep.staffPayment.option === StaffPaymentOptions.NO_FEE) }]
-      : []
+    let filingData: Array<FilingDataIF> = []
+    if (this.getFilingData) {
+      filingData = cloneDeep(this.getFilingData)
+      if (this.getEntityType === CorpTypeCd.COOP && this.getFilingData.length) {
```

Comment:
Change to `length > 0` to be explicit.

Explanation: you are not checking for general validity (truthiness) but specifically if there is at least 1 array element.

Reply:
ok!